fragments/k8s/cdk-aws/bundle.yaml

```diff
@@ -0,0 +1,76 @@
+# This is an incomplete bundle fragment. Do not attempt to deploy.
+description: |-
+    A highly-available, production-grade Kubernetes cluster designed to run on AWS.
+series: bionic
+services:
+  "kubernetes-master":
```

Comment:
I believe the point of fragments was to not repeat the same bungle fragment again.

If we are to do this with fragments i would expect a single folder under fragments called integrators that would hold only the integration charm. We would have one such integration charm per cloud provider.

The functionality of fragments is included in bundle overlays so perhapse it would have been better if we moved to overlays for this project.